pkg/converter/converter.go

```diff
@@ -65,6 +65,33 @@ func composeServiceVolumesToK8s(
 	return volumes, volumeMounts
 }
 
+func composeServiceTmpfsToK8s(
+	container string,
+	paths []string,
+) (map[string]core.Volume, []core.VolumeMount) {
+	volumeMounts := []core.VolumeMount{}
+	volumes := make(map[string]core.Volume)
+
+	for _, path := range paths {
+		name := fmt.Sprintf("%s-tmpfs-%s", container, strings.ReplaceAll(path, "/", "-"))
```

Comment:
I don't think that's good enough. Unix paths can contain a lot of special characters, k8s resource names are more restrictive, I think. A better way to deal with this would be to replace all illegal characters by dashes, then collapse subsequent dashes into one, and then remove dashes at start and beginning. Also think about lower/uppercase, not sure e.g. if you can have a k8s resource "foo" and another called "Foo" at the same time, so maybe you need to lowercase everything as well.

Reply:
It could also happen that after all the special chars removing and dashes collapsing we have nothing left really. In that case it would be better to just hash the original string and take a few characters from the hex string. I think k8ify already has code for that...

Reply:
Check out stringutils.go -> SanitizeWithMinLength